src/main/java/bio/terra/cli/command/workspace/SetProperty.java

```diff
@@ -0,0 +1,35 @@
+package bio.terra.cli.command.workspace;
+
+import bio.terra.cli.businessobject.Context;
+import bio.terra.cli.businessobject.Workspace;
+import bio.terra.cli.command.shared.BaseCommand;
+import bio.terra.cli.command.shared.options.Format;
+import bio.terra.cli.command.shared.options.WorkspaceOverride;
+import bio.terra.cli.command.shared.options.WorkspaceProperties;
+import bio.terra.cli.serialization.userfacing.UFWorkspace;
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+
+/** This class corresponds to the third-level "terra workspace set-property" command. */
+@Command(name = "set-property", description = "Set the workspace properties.")
```

Comment:
// This is set-property instead of add-property because this can be used to 1) Add property 2) Update existing property.